tests/openwrt/test_backend.py

```diff
@@ -19,6 +47,37 @@ def test_config_copy(self):
         o.validate()
         self.assertDictEqual(config, {'interfaces': []})
 
+    def test_plugin(self):
+        config = {
+            "chilli": {
+                'disabled': 1,
+                'uamsecret': 'asd123fghj'
+            },
+        }
+        OpenWrt.add_plugins([CoovaPlugin()])
```

Comment:
keep in mind that once you do this, the plugin will be enabled during all the tests that are called afterwards (even other test classes), this may have unintended consequences in the future, I think you should do two things:

- subclass the `OpenWrt` backend here in this file, redefine the `schema` and `converters` attributes to explicitly use a copy (use `deepcopy`) of `OpenWrt.schema` and `OpenWrt.converters`, then use the subclass here in the tests
- add a `tearDown` method which cleans up any additional plugin every time a test method terminates execution